<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IndustrialAnalogOutV2</class>
 <widget class="QWidget" name="IndustrialAnalogOutV2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>673</width>
    <height>177</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="checkbox_enable">
       <property name="text">
        <string>Enable Output</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radio_voltage">
       <property name="text">
        <string>Voltage</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radio_current">
       <property name="text">
        <string>Current</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_voltage" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_voltage">
        <property name="text">
         <string>Output Voltage [mV]:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="slider_voltage">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
        <property name="pageStep">
         <number>1000</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spin_voltage">
        <property name="maximum">
         <number>10000</number>
        </property>
        <property name="singleStep">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_voltage_range">
        <property name="text">
         <string>Range:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="box_voltage_range">
        <item>
         <property name="text">
          <string>0 - 5 V</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>0 - 10 V</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_current" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_current">
        <property name="text">
         <string>Output Current [ÂµA]:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="slider_current">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimum">
         <number>4000</number>
        </property>
        <property name="maximum">
         <number>20000</number>
        </property>
        <property name="pageStep">
         <number>1000</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spin_current">
        <property name="minimum">
         <number>4000</number>
        </property>
        <property name="maximum">
         <number>20000</number>
        </property>
        <property name="singleStep">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_current_range">
        <property name="text">
         <string>Range:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="box_current_range">
        <item>
         <property name="text">
          <string>4 - 20 mA</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>0 - 20 mA</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>0 - 24 mA</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
